.github/workflows/test.yml

```diff
@@ -106,19 +159,22 @@ jobs:
         run: docker run -t --rm --privileged test-integration ./hack/test-integration.sh -test.only-flaky=true
 
   test-integration-ipv6:
-    timeout-minutes: 60
+    needs: build-dependencies
+    timeout-minutes: 10
     name: ipv6 | ${{ matrix.containerd }} | ${{ matrix.ubuntu }}
     runs-on: "ubuntu-${{ matrix.ubuntu }}"
     strategy:
       fail-fast: false
       matrix:
-        # ubuntu-20.04: cgroup v1, ubuntu-22.04 and later: cgroup v2
         include:
           - ubuntu: 24.04
             containerd: v2.0.0-rc.5
+            arch: amd64
     env:
-      UBUNTU_VERSION: "${{ matrix.ubuntu }}"
       CONTAINERD_VERSION: "${{ matrix.containerd }}"
+      ARCH: "${{ matrix.arch }}"
+      RUNNER: "ubuntu-${{ matrix.ubuntu }}"
```

Comment:
Is this used in the test scripts?

Reply:
No, you are right, it is useless.
I was using it previously as part of the cache-key, but have since stopped.

Will remove.
Thanks!